web_dashboard_tile/__manifest__.py

```diff
@@ -0,0 +1,33 @@
+# © 2010-2013 OpenERP s.a. (<http://openerp.com>).
+# © 2014 initOS GmbH & Co. KG (<http://www.initos.com>).
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+{
+    "name": "Overview Dashboard (Tiles)",
+    "summary": "Add Overview Dashboards with Tiles",
+    "version": "12.0.1.0.0",
+    "depends": ["web", "board", "mail", "web_widget_color"],
+    "author": "initOS GmbH & Co. KG, "
+    "GRAP, "
+    "Odoo Community Association (OCA)",
+    "maintainers": ["legalsylvain"],
+    "category": "web",
+    "license": "AGPL-3",
+    "contributors": [
+        "initOS GmbH & Co. KG",
+        "GRAP",
+        "Iván Todorovich <ivan.todorovich@gmail.com>",
+    ],
```

Comment:
Is this key even used? :thinking: I guess we should move this to `CONTRIBUTORS.rst`

Reply:
I put values in ``authors`` keys.